docs/examples/extension.ipynb

```diff
@@ -11,14 +11,14 @@
    "cell_type": "markdown",
```

Comment:
<p>I'd keep to Pythonic naming conventions and call the class something like <code>PowerLawWithExpCut</code>.  Also, why all the space between <code>def _p</code>  and the contents of the function?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/halomod/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>